Projects/App/Sources/RootCore.swift

```diff
@@ -8,27 +8,36 @@
 
 import ComposableArchitecture
 import Foundation
-import Services
+import KakaoLogin
 
 @Reducer
 public struct RootCore {
+  @ObservableState
   public struct State: Equatable {
-
-    public init() {}
+    public var login: LoginCore.State = LoginCore.State()
   }
-
+  
   public enum Action {
+    case login(LoginCore.Action)
     case onOpenURL(URL)
   }
   
   @Dependency(\.kakaoLoginClient) var kakaoLoginClient
-
+  
   public var body: some Reducer<State, Action> {
+    Scope(state: \.login,
+          action: \.login) {
+      LoginCore()
+    }
+    
     Reduce { state, action in
       switch action {
+      case .login(_):
```

Comment:
P3:
```suggestion
      case .login:
```
연관값을 사용하지 않는다면 더 간단하게 선언하는 것도 가독성 측면에선 좋을 수 있습니다